documentation/documentation/docs/projectform/project_upload_form.md

```diff
@@ -0,0 +1,33 @@
+# Project upload guide
+A teacher can upload a new project of his course by going to the url ```projects/create``` where he will need to fill in data corresponding to the project he wants to upload.
```

Comment:
his -> their

Reply:
he -> they (x2)
fill in -> fill out

Reply:
https://github.com/SELab-2/UGent-3/pull/250/commits/c2af2565e6508f3e8caa588f821307480a79c078